Linux build issue with v0.15.0.5

Hi,

I've followed the linux build instructions but always keep getting this error message:

[ 87%] Building CXX object tests/unit_tests/CMakeFiles/unit_tests.dir/hmac_keccak.cpp.o
In file included from /usr/include/gtest/gtest.h:62,
                 from /home/xmr/Downloads/Temp/monero/tests/unit_tests/hmac_keccak.cpp:29:
/home/xmr/Downloads/Temp/monero/tests/unit_tests/hmac_keccak.cpp:126:1: error: static assertion failed: test_name must not be empty
  126 | TEST(keccak_hmac, )
      | ^~~~
make[3]: *** [tests/unit_tests/CMakeFiles/unit_tests.dir/build.make:388: tests/unit_tests/CMakeFiles/unit_tests.dir/hmac_keccak.cpp.o] Error 1
make[3]: Leaving directory '/home/xmr/Downloads/Temp/monero/build/Linux/release-v0.15/release'
make[2]: *** [CMakeFiles/Makefile2:4759: tests/unit_tests/CMakeFiles/unit_tests.dir/all] Error 2
make[2]: Leaving directory '/home/xmr/Downloads/Temp/monero/build/Linux/release-v0.15/release'
make[1]: *** [Makefile:141: all] Error 2
make[1]: Leaving directory '/home/xmr/Downloads/Temp/monero/build/Linux/release-v0.15/release'
make: *** [Makefile:103: release-all] Error 2

Please can someone assist.

Thanks

--- a/tests/unit_tests/hmac_keccak.cpp
+++ b/tests/unit_tests/hmac_keccak.cpp
@@ -123,7 +123,7 @@ static void test_keccak_hmac(const size_t * chunks)
 }
 
 
-TEST(keccak_hmac, )
+TEST(keccak_hmac, nullptr)
 {
   test_keccak_hmac(nullptr);
 }

--- a/tests/unit_tests/keccak.cpp
+++ b/tests/unit_tests/keccak.cpp
@@ -54,10 +54,6 @@ extern "C" {
   keccak_finish(&ctx, md1); \
   ASSERT_EQ(memcmp(md0, md1, 32), 0);
 
-TEST(keccak, )
-{
-}
-
 TEST(keccak, 0_and_0)
 {
   static const size_t chunks[] = {0};

